src/python/espressomd/cellsystem.pxd

```diff
@@ -39,6 +39,11 @@ cdef extern from "CellStructureType.hpp":
         CELL_STRUCTURE_REGULAR "CellStructureType::CELL_STRUCTURE_REGULAR"
         CELL_STRUCTURE_NSQUARE "CellStructureType::CELL_STRUCTURE_NSQUARE"
 
+cdef extern from "particle_data.hpp":
+    ctypedef struct particle "Particle"
+    const particle & get_particle_data(int pid) except +
+
+
```

Comment:
```suggestion
```

Note: each .pyx file includes all .pxd files. This struct and function are already declared in `particle_data.pxd`, so it's not necessary to include them here again.